extensions/wikia/PortableInfobox/tests/nodes/NodeDataTest.php

```diff
@@ -46,11 +46,12 @@ public function sourceDataProvider() {
 	 * @param $params
 	 * @param $expected
 	 */
-	public function testSourceLabel( $markup, $params, $expected ) {
-		$node = \Wikia\PortableInfobox\Parser\Nodes\NodeFactory::newFromXML( $markup, $params );
-
-		$this->assertEquals( $expected, $node->getSourceLabel() );
-	}
+	// TODO
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)